pkg/registry/defaults.go

```diff
@@ -167,3 +194,13 @@ func IsHeadlessEnabled(cr *registryv1alpha1.DevfileRegistry) bool {
 	}
 	return DefaultDevfileRegistryHeadlessEnabled
 }
+
+func getDevfileRegistrySpecContainer(quantity string, defaultValue string) resource.Quantity {
```

Comment:
We should include testing on the abstract private function too.

Reply:
Done! I've added test cases for the `getDevfileRegistrySpecContainer` func :)